tck/pom.xml

```diff
@@ -22,7 +22,7 @@
     <parent>
         <groupId>org.eclipse.ee4j</groupId>
         <artifactId>project</artifactId>

```

Comment:
Not that it should matter much but should the snapshot version also change at https://github.com/jakartaee/faces/blob/bd536e462adbd23ba65b0d9de2d738f0ba9095b1/tck/pom.xml#L31?

From `<version>4.0.3-SNAPSHOT</version>` to `<version>4.0.4-SNAPSHOT</version>`?

Reply:
It is done